modules/ROOT/pages/design-migrating-ms.adoc

```diff
@@ -0,0 +1,108 @@
+= Migrate to the Version 2 of the Mocking Service
+
+The mocking service lets you test endpoints that are defined in an API specification. The advantage to using the mocking service is that you do not need to implement the API before testing. To test an endpoint, the mocking service simulates calls to its methods.
+
+For example, when you are designing an API specification in API Designer, you can simulate a call to an endpoint’s GET method to see whether the response body for the 200 status code is as you want it. As a further example, if you are evaluating whether you want to implement an API specification that is in Exchange, you can simulate a call to an endpoint’s GET method to find out what the response body looks like.
+
+[IMPORTANT]
+====
+After January 31, 2020, version 1 of the mocking service will no longer send responses to requests. If you are using a third-party tool to send requests, the mocking service will return the HTTP status code 301 and provide a link to this document.
+====
+
+== How to Migrate an API-Specification Project to Version 2 of the Mocking Service
+
+If an API-specification project is older than January 10, 2019 and the mocking service was switched on before that date, follow these steps to migrate to version 2 of the mocking service:
+
+. In the notification that appears in the top-right corner after you open the project, click *Upgrade*. The text of the notification reads “A new mocking service is available. Upgrading takes just seconds.”
```

Comment:
I think it's not necessary to quote the text of the notification. If the docs want to convey the same message, it should do so naturally.

Reply:
Yeah, I was on the fence with that, thinking I needed to add some way of identifying the notification, so that users could feel certain that they were clicking the right one. I know that there is only one, but users won't necessarily know that there is only one notification in APID. They might assume that there could be others for other purposes.
Removed.